### PR TITLE
feat: add initial lineage yamls

### PR DESCRIPTION
**Pull request by corneliusroemer**
For each lineage, create one YAML file that summarises that contains essential information.

Add tests to validate that the schema is correct.

Auto-generate a single file output that contains all the information for down-stream users.

### CODE REVIEW

**Review by rneher — APPROVED** [review]
LGTM. But you could add some tooling in addition -- maybe this uncovers potential design flaws.

File: lineages/A.2.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=../schemas/lineage_schema.yml
+name: B.1.1
```
Comment (AngieHinrichs) [inline lineages/A.2.yml:2]: This is the only one whose name doesn't match it's filename.

Comment (emmahodcroft) [reply lineages/A.2.yml:2]: Good catch Angie - looks like a copy/paste bug probably! @corneliusroemer

**Comment by emmahodcroft** [comment]
Should we not also have a definition of B.1 as well? I'm not seeing the file.

File: lineages/A.2.yml (hunk 1 above)
Comment (corneliusroemer) [reply lineages/A.2.yml:2]: Oops - sorry, thanks for the catch.

In the process of writing some CI scripts that would catch this in the future.

File: auto-generated/alias_key.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "A": "A",
```
Comment (chaoran-chen) [inline auto-generated/alias_key.json:2]: Does this line make sense? It is different than in [pango-designations](https://github.com/cov-lineages/pango-designation/blob/master/pango_designation/alias_key.json) and might cause a resolver implemented for Pango to end up in an infinite loop.

File: auto-generated/lineages.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    "reference_sequences": [
+      {
+        "accession": "ON563414",
+        "isolate": "MPXV_USA_2022_MA001",
+        "source": "genbank"
+      }
+    ],
+    "unaliased_name": "A.1.1.1"
+  }
+]
```
Comment (chaoran-chen) [inline auto-generated/lineages.json:230]: I believe that it's better to have a newline at the end of the file.

**Review by chaoran-chen — COMMENTED** [review]
Am I understanding correctly that tools are encouraged to read `auto-generated/lineages.json` and not the YAML files? Then, I think that it would be great if the json file contains a version field. I.e., I would reshape the file to:

```
{
  version: X,
  lineages: [...]
}
```

I think that it is in most cases a good idea to start a JSON file with an object rather than an array because it allows one to add further top-level properties in the future.

Regarding version numbers, there's the choice between a simple integer or using major/minor versions. Docker-compose uses major/minor versions and I think that's quite nice (https://docs.docker.com/compose/compose-file/compose-versioning/#version-3). Non-breaking changes would then increase the minor version and breaking changes the major version.

We'd then of course also need a changelog file that explains the changes to a new version.

File: auto-generated/lineages.json (hunk 3 above)
Comment (corneliusroemer) [reply auto-generated/lineages.json:230]: Thanks! Will insert

File: auto-generated/alias_key.json (hunk 2 above)
Comment (corneliusroemer) [reply auto-generated/alias_key.json:2]: Fair point - I'll stick t opango conventions then :)

**Pull request merged**